_pages/dev/backend_communication/loading-scopes.md

```diff
@@ -187,3 +187,34 @@ The `reloadOn` configuration allows you to reload a product when a specific even
 ```
 
 In this example, the `details` scope is reloaded when `MyEvent` is emitted.
+
+## Support Scopes for ProductSearch Endpoint
+
+`ProductSearch` endpoint can also be configured to use scopes with limit function now. Scopes in `ProductSearch` don't support `Merging` and `Inclusion` features yet which should be considered when introduce new scope for `ProductSearch`.
+The following example shows the improvement change:
+
+```ts
+{
+  // before
+  backend: {
+    occ: {
+      endpoints: {
+        productSearch:'products/search?XXXXXX',
+      },
+    },
+  },
+  //after
+  backend: {
+    occ: {
+      endpoints: {
+        productSearch: {
+          default:'products/search?XXXXXX',
+          carousel:'products/search?XXX',
+          carouselMinimal:'products/search?XX',
+        },
+      },
+    },
+  }    
+}
+```
+With this configuration, you can request multi product data with ProductSearch endpoint but not invoke Product endpoint one by one. For example, after enable feature toggle `useProductCarouselBatchApi`, all Product Carousel Component will use ProductSearch endpoint with `carouselMinimal` scope to get product data which improve performance.
```

Comment:
I am wondering about the meaning of this sentence:

"you can request multi product data with ProductSearch endpoint but not invoke Product endpoint one by one"

Does it mean that you can request a more limited set of product data, such as defined by `carouselMinimal`, and that means you don't have to invoke all of the existing `products` fields, as you would if you used the `default` scope (or if you used the older `productSearch` endpoint)?

Or does "not invoke Product endpoint one by one" have something to do with `useProductCarouselBatchApi`? To me, the name `useProductCarouselBatchApi` implies there is a batch process involved, where maybe you can request all the product data for the items in a carousel with a single call, rather than having to make a separate request for each item in the carousel, but... All I can see for sure is that enabling `useProductCarouselBatchApi` means that product carousel components will use the `carouselMinimal` scope, but I don't see exactly how "batch" comes into play..... 🤷 🙂